### PR TITLE
add spaces immediately inside brackets of group by legends

### PR DESCRIPTION
**Pull request by pitr**
also, show metric name if no alias is set

### CODE REVIEW

File: src/core/response/series_name_builder.ts [hunk 1]
```diff
@@ -8,17 +8,14 @@ export class SeriesNameBuilder {
             tagGroupBysValues = this.getTagGroupBys(tagGroupBys),
             valueGroupBysValues = this.getValueGroupBys(groupBys),
             timeGroupBysValues = this.getTimeGroupBys(groupBys);
-        if (_.isEmpty(alias)) {
-            // return "tag1=value1 tag2=value2" or simply "metricName"
```
Comment (pitr) [inline src/core/response/series_name_builder.ts:12]: I am removing this case as this is not what Grafana3 does

File: src/core/response/series_name_builder.ts [hunk 2]
```diff
@@ -8,17 +8,14 @@ export class SeriesNameBuilder {
...
+        if (alias.indexOf("$_") >= 0 ) {
             // evaluate expressions in alias
             return this.buildAlias(alias, tagGroupBys, valueGroupBysValues, timeGroupBysValues);
         } else {
-            // return "alias (tag1=value1 tag2=value2)" or simply "alias"
+            // "alias ( tag1=value1 tag2=value2 )" or simply "alias"
             const groupBysName = this.buildDefault(tagGroupBysValues, valueGroupBysValues, timeGroupBysValues);
-            return _.isEmpty(groupBysName) ? alias : alias + " (" + groupBysName + ")";
+            const prefix = _.isEmpty(alias) ? metricName : alias;
+            return _.isEmpty(groupBysName) ? prefix : prefix + " ( " + groupBysName + " )";
```
Comment (pitr) [inline src/core/response/series_name_builder.ts:18]: notice spaces immediately inside brackets

**Comment by lfroment0** [comment]
:+1:

**Comment by pitr** [comment]
👍

**Pull request merged**